### ISSUE TITLE
Not catching Pokemon nor looting pokestops

### ISSUE THREAD

**Issue by ghost** [disdain13/PokeRoadie#211]
[2016-09-22 - 08.03.53.txt](https://github.com/disdain13/PokeRoadie/files/487442/2016-09-22.-.08.03.53.txt)

Everytime I use the bot, he catches a few pokemon in the first minutes and after that he just keep moving around not catching any pokemon nor looting any pokestop, he visits then but does not loot.
I'm not sure if it's a bug or if it's some wrong configuration. Using latest version.
Btw, congrats for the good job! 👍

**Comment by Magyar00** [comment]
It could be a soft ban => did you teleport to a location far from you usually play ? Did you increase your max speed ?

**Comment by ghost** [comment]
No, speed is default and I have always used the same location coord

**Comment by ai3gtmc** [comment]
This happens when to me when my buddy pokemon finds a candy. For some reason it won't do anything other than travel. Probably the same thing happening to you. Open the POGO on your phone and see if it found any candy.

**Comment by BoomEX-UK** [comment]
line 330 of the log:
[08:18:31] Distance to target location: 33,52 meters. Will take 1,83 seconds driving at 66kmh
[08:18:34] SOFT BAN Detected a soft ban, let's chill out a moment.

**Comment by Vlambal** [comment]
Having the same a similar problem with the latest bot  not catching any pokemon but still doing about 2 pokestops in my area, must say that the incubator setting is working wonderful
 any ideas about that and how do I change my device id to use my I phone to clear some of my pokemon
